webcommon/web.client.samples/build.xml

```diff
@@ -21,13 +21,4 @@
 -->
 <project basedir="." default="build" name="webcommon/web.client.samples">
     <description>Builds, tests, and runs the project org.netbeans.modules.web.client.samples</description>
-    <import file="../../nbbuild/templates/projectized.xml"/>
```

Comment:
line 24 needs to stay (this is the inclusion of the normal netbeans build harness)